src/app/doc_exporter.cpp

```diff
@@ -599,8 +602,17 @@ void DocExporter::captureSamples(Samples& samples)
           sample.setTrimmedBounds(frameBounds = gfx::Rect(0, 0, 0, 0));
         }
 
-        if (m_trimCels)
+        if (m_trimCels) {
+          if (m_trimByGrid) {
+            auto& docPref = Preferences::instance().document(doc);
+            gfx::Point startingPoint(frameBounds.x, frameBounds.y);
+            gfx::Point endingPoint(frameBounds.x + frameBounds.w, frameBounds.y + frameBounds.h);
+            gfx::Point posTopLeft = snap_to_grid(docPref.grid.bounds(), startingPoint, PreferSnapTo::FloorGrid);
```

Comment:
Too long lines here, and we can use the `origin()` and `point2()` member functions from `gfx::Rect` as the starting/ending points:
```c++
            gfx::Point posTopLeft =
              snap_to_grid(docPref.grid.bounds(),
                           frameBounds.origin(),
                           PreferSnapTo::FloorGrid);
            gfx::Point posBottomRight =
              snap_to_grid(docPref.grid.bounds(),
                           frameBounds.point2(),
                           PreferSnapTo::CeilGrid);
            frameBounds = gfx::Rect(posTopLeft, posBottomRight);
```

Reply:
Done